docs/dev/framework/dca/_index.md

```diff
@@ -159,38 +153,6 @@ class NewsOnsubmitCallbackListener
 }
 ```
 
-
-### Using the PHP array configuration
-
-This is the old way of using DCA callbacks prior to Contao **4.7**. The table
-`tl_content` and target definition `fields.module.options` translates to a PHP
-array configuration in the following way for example:
-
-```php
-// contao/dca/tl_content.php
-$GLOBALS['TL_DCA']['tl_content']['fields']['module']['options_callback'] = [
-    \App\DataContainer\ContentListener::class, 'onModuleOptionsCallback'
-];
-```
-
-Note that you have to add the suffix `_callback` to the last key - _except_ for
-these callbacks:
-
-* `fields.field.wizard`
-* `fields.field.xlabel`
-* `list.sorting.panel_callback.subpanel`
-
-Also note that the above example is a _singular_ callback, i.e. it only
-supports _one_ subscriber. For callbacks that support multiple callback
-subscribers you'll have to append the callable:
-
-```php
-// contao/dca/tl_content.php
-$GLOBALS['TL_DCA']['tl_content']['config']['onload_callback'][] = [
-    \App\DataContainer\ContentListener::class, 'onLoadCallback'
-];
-```
-
```

Comment:
This must not be removed. You still need to manually register a callback via the DCA configuration when re-using a service from another bundle (see contao/newsletter-bundle for example).